src/main/java/com/compatibility/tinkers/ToolBrokenCheck.java

```diff
@@ -0,0 +1,44 @@
+package com.compatibility.tinkers;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import org.jetbrains.annotations.Nullable;
+
+/**
+  * This class is to store a check to see
+  * if a tinker's tool is broken
+  */
+public class ToolBrokenCheck
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this class "final" or add a public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2974)